### PR TITLE
Fix function suffix grammar. Fix issue #214

### PR DESCRIPTION
**Pull request by clementval**
Fix for #214

### CODE REVIEW

**Comment by clementval** [comment]
@shingo-s Sorry, I didn't check that point. I'll fix this.

**Comment by clementval** [comment]
@shingo-s I updated the grammar to avoid multiple rules to accept empty.

File: F-FrontEnd/src/F95-parser.y [hunk 1]
```diff
@@ -842,20 +843,30 @@ program_name:   /* null */
         | IDENTIFIER
         ;
 
+func_result:
+        RESULT '(' name ')'
+        { $$ = $3; printf("FUNC RESULT OK\n"); }
```
Comment (shingo-s) [inline F-FrontEnd/src/F95-parser.y]: Please remove debug print

Comment (clementval) [inline F-FrontEnd/src/F95-parser.y]: Sorry :( done.

**Comment by shingo-s** [comment]
LGTM👍

**Pull request merged**